File: airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/PostgresSourceSSLFullCertificateAcceptanceTest.java
/*
 * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.integrations.io.airbyte.integration_tests.sources;

import com.google.common.collect.ImmutableMap;

public class PostgresSourceSSLFullCertificateAcceptanceTest extends AbstractPostgresSourceSSLCertificateAcceptanceTest {

  @Override
  public ImmutableMap getCertificateConfiguration() {
    return ImmutableMap.builder()
        .put("mode", "verify-ca")
        .put("ca_certificate", certs.getCaCertificate())
        .put("client_certificate", certs.getClientCertificate())
        .put("client_key", certs.getClientKey())
        .put("client_key_password", PASSWORD)
        .build();
  }

}

File: airbyte-integrations/connectors/source-postgres/src/test/kotlin/io/airbyte/integrations/source/postgres/legacy/PostgresSourceSSLFullCertificateAcceptanceTest.kt
/*
 * Copyright (c) 2026 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.integrations.source.postgres.legacy

import com.google.common.collect.ImmutableMap

class PostgresSourceSSLFullCertificateAcceptanceTest :
    AbstractPostgresSourceSSLCertificateAcceptanceTest() {
    override val certificateConfiguration: MutableMap<Any?, Any?>
        get() =
            ImmutableMap.builder<Any?, Any?>()
                .put("mode", "verify-ca")
                .put("ca_certificate", testdb.certificates.caCertificate)
                .put("client_certificate", testdb.certificates.clientCertificate)
                .put("client_key", testdb.certificates.clientKey)
                .put("client_key_password", PASSWORD)
                .build()
}
